From 12ae1a9a62c2c94af44f55b03575ab8806bd22ee Mon Sep 17 00:00:00 2001
From: Milind Changire <mchangir@redhat.com>
Date: Mon, 23 Apr 2018 13:16:30 +0530
Subject: [PATCH 25/52] build: make RHGS version available for server

Make /usr/share/glusterfs/release available for gluserfs-server package.
This file contains the RHGS release number for the release.

Label: DOWNSTREAM ONLY

Change-Id: I7485f77cfb8ca7f0f8363a20124900ae9ae8a528
Signed-off-by: Milind Changire <mchangir@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/137139
Tested-by: RHGS Build Bot <nigelb@redhat.com>
Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
---
 glusterfs.spec.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index db50b8e..bdb47ba 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -862,6 +862,10 @@ install -p -m 0744 -D extras/command-completion/gluster.bash \
     %{buildroot}%{_sysconfdir}/bash_completion.d/gluster
 %endif
 
+%if ( 0%{!?_without_server:1} )
+echo "RHGS 3.5" > %{buildroot}%{_datadir}/glusterfs/release
+%endif
+
 %clean
 rm -rf %{buildroot}
 
@@ -1452,6 +1456,7 @@ exit 0
 
 # Extra utility script
 %dir %{_libexecdir}/glusterfs
+     %{_datadir}/glusterfs/release
 %dir %{_datadir}/glusterfs/scripts
      %{_datadir}/glusterfs/scripts/stop-all-gluster-processes.sh
 %if ( 0%{?_with_systemd:1} )
-- 
1.8.3.1

